### PR TITLE
Update chokidar, anymatch, source-map. Simplify package.json scripts. Remove setup.js

### PR DESCRIPTION
**Pull request by yavorsky**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,18 +27,16 @@
...
   },
   "preferGlobal": true,
   "dependencies": {
-    "anymatch": "~1.0.0",
+    "anymatch": "~1.1.0",
     "anysort": "~1.0.0",
     "async-each": "~0.1.2",
     "async-waterfall": "~0.1.2",
-    "chokidar": "~0.12.0",
+    "chokidar": "~1.0.0",
```
Comment (es128) [inline package.json]: does this resolve to the latest rc?

Comment (es128) [inline package.json]: yes, apparently it does

Comment (yavorsky) [inline package.json]: @es128 :+1:

File: package.json [hunk 2]
```diff
@@ -27,18 +27,16 @@
     "brunch": "./bin/brunch"
   },
   "scripts": {
-    "prepublish": "node setup.js prepublish",
-    "postpublish": "node setup.js postpublish",
-    "test": "node setup.js test",
-    "postinstall": "node setup.js postinstall"
+    "prepublish": "coffee -o lib/ src/",
```
Comment (paulmillr) [inline package.json]: i'm not sure whether this is a cross-platform solution. can someone check for win32 support?

**Pull request merged**